owncloudData/src/main/java/com/owncloud/android/data/webfinger/repository/OCWebfingerRepository.kt

```diff
@@ -19,11 +19,33 @@ package com.owncloud.android.data.webfinger.repository
 
 import com.owncloud.android.data.webfinger.datasources.WebfingerRemoteDatasource
 import com.owncloud.android.domain.webfinger.WebfingerRepository
+import com.owncloud.android.domain.webfinger.model.WebfingerRel
 
 class OCWebfingerRepository(
     private val webfingerRemoteDatasource: WebfingerRemoteDatasource,
 ) : WebfingerRepository {
 
-    override fun getJRDFromWebFingerHost(server: String, resource: String): String =
-        webfingerRemoteDatasource.getJRDFromWebFingerHost(lookupServer = server, username = resource)
+    override fun getInstancesFromWebFinger(
```

Comment:
I see in some methods or classes you write `WebFinger` and in others `Webfinger`... This is not important at all but I'd try to unify it to be consistent 👍